src/ProjectFileManager.h

```diff
@@ -81,13 +82,17 @@ class AUDACITY_DLL_API ProjectFileManager final
 
    static bool IsAlreadyOpen(const FilePath &projPathName);
 
+   //! A function that returns a project to use for opening a file; argument is true if opening a project file
+   using ProjectChooser = std::function<AudacityProject&(bool)>;
```

Comment:
Using the same name as the class causes confusion.  Add suffix of Fn, perhaps?

Reply:
ok